modules/osd-storage-pv-aws-config-account.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/
```

Comment:
This should point to the assembly `storage/persistent_storage/osd-persistent-storage-aws.adoc`

Also applies to all other modules